definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-general-referral-nonprod.json

```diff
@@ -627,14 +627,14 @@
     "CaseTypeID": "DIVORCE",
     "CaseFieldID": "FeeAmountWithoutNotice",
     "UserRole": "caseworker-divorce-courtadmin_beta",
-    "CRUD": "CRUD"
+    "CRUD": "CRU"
```

Comment:
can you please make sure you are able to trigger event that is using this field more than once and prove it still works?